### ISSUE TITLE
Sugest: Sembast Editor

### ISSUE THREAD

**Issue by rafaband** [tekartik/sembast.dart#209]
I was thinking about it.... We have a nice feature that allow us to export the database file and import it?! Maybe it'll be nice to have one editor.

We build our app, edit the data, creating some default values, that we can import on the flutter build.

H2 has one console, SQLite has many editors. Is it possible?! If so I 'd like to participate on this feature

**Comment by alextekartik** [comment]
Good idea. I started one but really tailored to my needs so likely not generic enough nor ready for primetime. I think one missing API to build an editor is the ability to get the list of stores and some meta information otherwise a generic editor won't be able to find all the data. Since a store don't really exist, maybe a method like `nonEmptyStoreNames` returning the list of store names that have at least one record could help.

**Comment by flutter-painter** [comment]
Hi,
Gread idea, same need here and also willing to contribute.
@alextekartik how did you manage the UI for your own editor ?
I was looking at **pluto_grid** for the UI in order to be able to edit data directly.
It would also be very handy to be able to run Dart/sembast queries


```dart
await intMapStoreFactory.store('shops').addAll(_database, [Shop(1,'coolShop')].map((s) =>s.toMap()).toList());
```
